OOModel/src/declarations/Declaration.cpp

```diff
@@ -27,6 +27,9 @@
 #include "Declaration.h"
 
 #include "ModelBase/src/nodes/TypedListDefinition.h"
+
+#include "expressions/Expression.h"
```

Comment:
You don't need to include the `Expression.h` both in the ".h" and the ".cpp" file. Since you need it in the ".h" file, just leave it there and remove it here (including the empty line before and after, to make the diff cleaner).